extensions/functions_datetime.yaml

```diff
@@ -265,3 +342,111 @@ scalar_functions:
           - name: y
             value: interval_year
         return: boolean
+  -
+    name: "assume_timezone"
+    description: Convert local time to UTC using given local time's timezone.
+    impls:
+      - args:
+          - name: x
+            value: timestamp
+          - name: timezone
+            value: string
+        return: timestamp_tz
+  -
+    name: "strptime"
+    description: Parse string into timestamp/date/time using provided format,
+      see https://man7.org/linux/man-pages/man3/strptime.3.html for reference.
+    impls:
+      - args:
+          - name: timestamp_string
+            value: string
+          - name: format
+            value: string
+          - name: timezone
+            value: string
+        return: timestamp_tz
+      - args:
+          - name: timestamp_string
+            value: string
+          - name: format
+            value: string
+        return: timestamp
```

Comment:
Any particular reason the parse direction is missing implementations for `date` and `time` types?

Reply:
Not really. Added.